java/token-requestor-client/src/main/java/no/bankaxept/epayment/client/tokenrequestor/TokenRequestorClient.java

```diff
@@ -44,6 +44,11 @@ public Flow.Publisher<RequestStatus> delete(String tokenId, String correlationId
                 httpResponse -> httpResponse.getStatus().toResponse());
     }
 
+    public Flow.Publisher<RequestStatus> delete(String tokenId, String correlationId, Map<String, List<String>> headers) {
+        return new MapOperator<>(baseClient.post(String.format(DELETION_URL, tokenId), new SinglePublisher<>("", executor), correlationId, headers),
+                httpResponse -> httpResponse.getStatus().toResponse());
+    }
```

Comment:
Cant use the `SimulationRequest` here?

Reply:
Um, no. When me and @thxmasj  talked about it last, we were lucky enough that all requests that need simulation header all had bodies. Any new thoughts now?

Reply:
If we stick with the convention we use for payments, that the first client request specifies all the simulation for the scenario, this should still work. The simulation header would then be transported with the enrolment request and not the deletion request. By sticking to that convention we can support much broader simulations than a per request simulation.